data/json/item_groups.json

```diff
@@ -823,6 +823,13 @@
       ["purse", 40],
       ["mbag", 20],
       ["fanny", 10],
+	  ["acoustic_guitar", 20],
```

Comment:
Hmm, I've heard about that, and that's why I *thought* I only used spaces for these.